pom.xml

```diff
@@ -67,8 +67,8 @@
             <!-- this can be overriden in child POMs to support specific SDK requirements -->
             <groupId>dev.openfeature</groupId>
             <artifactId>sdk</artifactId>
-            <!-- 1.12 <= v < 2.0 -->
-            <version>[1.12,2.0)</version>
+            <!-- 1.12 <= v < 2.0 (excluding 2.0 pre-releases)-->
+            <version>[1.14,1.99999)</version>
```

Comment:
`1.99999` doesn't include pre-releases... `<2.0` does, unfortunately. This is unrelated but safer if we start releasing `2.0.0-alpha`, etc.